node/config/types.go

```diff
@@ -207,7 +207,14 @@ type DealmakingConfig struct {
 	// another concurrent download is allowed to start).
 	HttpTransferStallTimeout Duration
 
-	BitswapPeerID          string
+	// The peed id used by booster-bitswap. To set, copy the value
+	// printed by running 'booster-bitswap init'. If this value is set,
+	// Boost will:
+	// - listen on bitswap protocols on its own peer id and forward them
+	// to booster bitswap
+	// - advertise bitswap records to the content indexer
+	// - list bitswap in available transports on the retrieval transport protocol
```

Comment:
```suggestion
	//   - listen on bitswap protocols on its own peer id and forward them
	//     to booster bitswap
	//   - advertise bitswap records to the content indexer
	//   - list bitswap in available transports on the retrieval transport protocol
```

I know the primary consumer of this is for config file docs and that there's already precedent in there of lists starting at the front, but I wonder if we shouldn't be adapting in here now to the 1.19 godoc rules for properly rendering lists? It'll show up nicely on godocs and also looks a bit clearer in text IMO.

take it or leave it